includes/class-contact.php

```diff
@@ -135,7 +135,11 @@ public function widget( $args, $instance ) {
 
 		if ( 'yes' === $instance['map']['value'] && ! empty( $fields['address']['value'] ) ) {
 
-			add_action( 'wp_footer', [ $this, 'defer_map_iframes_js' ] );
+			if ( $this->defer_map_iframes && ! has_action( 'wp_footer', [ $this, 'defer_map_iframes_js' ] ) ) {
```

Comment:
@fjarrett It seems like you are right. It does however call add_action multiple times though. I think it's safer to check if the action wasn't added before so we don't add it over and over again on all instances of the widgets.